README.md

```diff
@@ -306,3 +306,17 @@ We also have a confluence page
 `https://nhsd-confluence.digital.nhs.uk/display/SPINE/CPM+Swagger+Docs`
 
 In time we will also have our spec uploaded to bloomreach via proxygen
+
+## SBOM
```

Comment:
Is it worth putting the full title along with this? SBOM (Service Bill of Materials). I'm likely to forget it if I haven't thought about it in a while, but if this is pretty common/obvious then just ignore me :)

Reply:
Very valid point! put it in now :)